.gitlab/ci/build_lassen.yml

```diff
@@ -0,0 +1,100 @@
+##############################################################################
+# Copyright (c) 2016-19, Lawrence Livermore National Security, LLC and Umpire
+# project contributors. See the COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (MIT)
+##############################################################################
+
+####
+# This is the share configuration of jobs for lassen
+
+.build_lassen:
+  variables:
+  tags:
+    - shell
+    - lassen
+  stage: build
+  extends: .build_blueos_3_ppc64le_ib_p9_script
+  except:
+    refs:
+      - /_lnone/
+    variables:
+      - $UMPIRE_CI_LASSEN == "OFF"
```

Comment:
Why is this variable set to `"OFF"`? I don't quite understand how these variables work.

Reply:
This is in fact a test for the except exclusion to apply.
It reads this like:
Jobs using `.build_lassen` property will be run `except` if _either_ ref contains `_lnone` _or_ `$UMPIRE_CI_LASSEN == "OFF")`.